src/epics/revisions.js

```diff
@@ -52,7 +52,8 @@ export const fetchRevisions = ( action$, store ) => (
 									data.revid,
 									new Revision( {
 										id: data.revid,
-										...data
+										...data,
+										commenthidden: 'commenthidden' in data
```

Comment:
@dayllanmaza I don't _think_ you should need this? If it's missing, Immutable will make it set to `undefined` which will fail your conditional in the component.